woocommerce-services.php

```diff
@@ -577,6 +588,11 @@ public function jetpack_on_plugins_loaded() {
 		public function on_plugins_loaded() {
 			$this->load_textdomain();
 
+			if ( apply_filters( 'wc_services_will_handle_coexistence_with_woo_shipping_and_woo_tax', false ) ) {
```

Comment:
[Non-blocking] I'm mentioning this with the intent of knowledge sharing the standard more than anything since the rest of the plugin doesn't really adhere to it anyway, but [WP standards wants us to document do_action and apply_filters](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#4-hooks-actions-and-filters) (IMO most importantly for the "since" tag to help determine backwards compatibility when using the hook)

```suggestion
			/**
			 * Allow third party logic to determine if this plugin should initiate its logic.
			 *
			 * The primary purpose here is to allow a smooth transition between the new Woo Shipping / Woo Tax plugins
			 * and WooCommerce Shipping & Tax (this plugin), by letting them take over all responsibilities if all three
			 * plugins are activated at the same time.
			 *
			 * @since {{next-release}}
			 * 
			 * @param bool $status The value will determine if we should initiate the plugins logic or not.
			 */
			if ( apply_filters( 'wc_services_will_handle_coexistence_with_woo_shipping_and_woo_tax', false ) ) {
```

Reply:
Very cool, thanks!